.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -18,13 +18,13 @@ Release candidate
 - [ ] Update changelog and open PR targeting a new `maint/<release version number>` branch
 - [ ] Update `ci-src-requirements.txt` if needed
 - [ ] Check MANIFEST and requirements are still up to date.
-- [ ] Update version in setup.py for the prerelease, open 2 PRs against maint/<release version number>
-       - Create a new branch from maint/<release version number>: `git checkout maint/<release version number>; git pull; git checkout -b call-it-anything-you-like`
-       - Set PRERELEASE to "rc1" and IS_RELEASED to true, commit, e.g. git commit -m "Set IS_RELEASED to true for prerelease <release version number>rc1"
-       - Open and merge a PR against maint/<release version number>
-       - Create a new branch from maint/<release version number>, Flip IS_RELEASED back to false, commit.
-       - Open and merge another PR against maint/<release version number>
-- [ ] Tag (annotated!) the release candidate on the commit where IS_RELEASED is set to true, e.g. git tag -a -m "Release candidate <release version number>rc1" <release version number>rc1 <commit-hash>
+- [ ] Update version in setup.py for the prerelease, open 2 PRs against `maint/<release version number>`
+  - [ ] Create a new branch from `maint/<release version number>`: `git checkout maint/<release version number>`; `git pull`; `git checkout -b call-it-anything-you-like`
+  - [ ] Set `PRERELEASE` to "rc1" and `IS_RELEASED` to true, commit, e.g. git commit -m "Set IS_RELEASED to true for prerelease <release version number>rc1"
+  - [ ] Open and merge a PR against `maint/<release version number>`
+  - [ ] Create a new branch from `maint/<release version number>`, Flip `IS_RELEASED` back to false, commit.
+  - [ ] Open and merge another PR against `maint/<release version number>`
+- [ ] Tag (annotated!) the release candidate on the commit where IS_RELEASED is set to true, e.g. `git tag -a -m "Release candidate <release version number>rc1" <release version number>rc1 <commit-hash>`
 - [ ] Push the tag to GitHub
 - [ ] Upload to PyPI
        - `git checkout <tag>`, `git clean -ffxd`, `python setup.py sdist`, `twine check dist/<...>.tar.gz`, `twine upload dist/<...>.tar.gz`
```

Comment:
The `Release blockers` section doesn't get rendered correctly because it's just `- [ ]`. It might be useful to add a few words there.